app/views/shelters/show.html.erb

```diff
@@ -8,3 +8,21 @@
 <%= link_to 'Update This Shelter', "/shelters/#{@shelter.id}/edit", class: 'button-left'%>
 <%= link_to 'Delete This Shelter', "/shelters/#{@shelter.id}", method: :delete, class: 'button-right' %>
 </div>
+</br>
+<section class= 'center-div'>
+  <% @shelter.reviews.each do |review| %>
+    <section id=reviews<%"#{review.id}"%>
+    </br>
+      <p> Review #<%=review.id%> </p>
+      <p><%= review.title%> </p>
+      <p><%= review.rating %> stars</p>
+      <p> <b> Review: </b> <%= review.content %></p>
+        <% if "#{review.image_url}".present? %>
```

Comment:
Let's check with an instructor about this on Thursday to see if we should leave this logic here or move it elsewhere.